apps/docs/src/app/core/component-docs/datetime-picker/examples/datetime-form-example/datetime-form-example.component.html

```diff
@@ -1,23 +1,28 @@
 <form [formGroup]="customForm" class="flex-form">
-    <div *ngIf="customForm">
-        <fd-datetime-picker formControlName="date"></fd-datetime-picker><br />
-        Touched: {{ customForm.controls.date.touched }}<br />
-        Dirty: {{ customForm.controls.date.dirty }}<br />
-        Valid: {{ customForm.controls.date.valid }}<br />
+
+    <div fd-form-item>
+        <label fd-form-label>Valid Date Picker</label>
+        <fd-datetime-picker [state]="'valid'" formControlName="date"></fd-datetime-picker>
+        <fd-form-message [type]="'success'">This is valid DateTimePicker</fd-form-message>
+        <br/>
+        Touched: {{ customForm.controls.date.touched }}<br/>
+        Dirty: {{ customForm.controls.date.dirty }}<br/>
+        Valid: {{ customForm.controls.date.valid }}<br/>
         Selected Date:
         {{ customForm.controls.date.value ? customForm.controls.date.value.toLocaleDateString() : 'null' }}
     </div>
-    <div>
-        <fd-datetime-picker formControlName="disabledDate"></fd-datetime-picker><br />
-        Touched: {{ customForm.controls.disabledDate.touched }}<br />
-        Dirty: {{ customForm.controls.disabledDate.dirty }}<br />
-        Valid: {{ customForm.controls.disabledDate.valid }}<br />
-        Disabled: {{ customForm.controls.disabledDate.disabled }}<br />
+    <br/><br/>
+    <div fd-form-item>
+        <label fd-form-label>Disabled Date Picker</label>
+        <fd-datetime-picker [state]="'information'" formControlName="disabledDate"></fd-datetime-picker>
```

Comment:
cursor still has hand on hover

Reply:
There is added `pointer-events: none` no disabled popovers.